src/main/java/dk/kb/labsapi/metadataFormats/IllustrationMetadata.java

```diff
@@ -24,19 +24,29 @@
  *     <li>x: height of illustration</li>
  * </ul>
  */
-public class IllustrationMetadata {
+public class IllustrationMetadata extends BasicMetadata {
     private static final Logger log = LoggerFactory.getLogger(IllustrationMetadata.class);
-    private String pageUUID;
-    private double pageWidth;
-    private double pageHeight;
     private String id;
     private double x;
     private double y;
     private double w;
     private double h;
     static final Pattern illustrationPattern = Pattern.compile("id=(\\S*),x=(\\d*),y=(\\d*),w=(\\d*),h=(\\d*),doms_aviser_page:uuid:(\\S*),(\\d*),(\\d*)");
 
-    public IllustrationMetadata(){
+    public IllustrationMetadata(String input){
```

Comment:
With this nice encapsulation of illustrations, maybe remove the intermediate String serialization/deserialization and let `ImageExport.getIllustrationsList` construct `IllustrationMetadata` directly?

Reply:
With the new constructor, `public IllustrationMetadata(String input){` is **only** used with unit tests. It should be removed (and the tests adjusted to use the new constructor.